src/shared/copy.c

```diff
@@ -878,6 +928,14 @@ static int fd_copy_regular(
                         return r;
         }
 
+        /* NB: fs-verity cannot be enabled when a writable file descriptor is outstanding.
+         * copy_fs_verity() may well re-open 'fdt' as O_RDONLY.  All code below this point
+         * needs to be able to work with a read-only file descriptor.
+         */
+        if (FLAGS_SET(copy_flags, COPY_FS_VERITY)) {
+                copy_fs_verity(fdf, &fdt);
```

Comment:
return code should be checked and logged (other than not supported)

Reply:
Right!  I totally missed this one, thanks.